lib/module.js

```diff
@@ -489,7 +489,9 @@ Module._initPaths = function() {
 
   var nodePath = process.env['NODE_PATH'];
   if (nodePath) {
-    paths = nodePath.split(path.delimiter).concat(paths);
+    paths = nodePath.split(path.delimiter).concat(paths).filter(function(path) {
+      return !!path
+    });
```

Comment:
Also, you could do

````js
return path !== '';
````
to make it more clear that the intend is to filter the empy string.